downstream/titles/hub/configuring-private-hub-rh-certified/master.adoc

```diff
@@ -1,33 +1,44 @@
+
 :imagesdir: images
 = Managing Red Hat Ansible Content Collections and Ansible Galaxy collections in automation hub
 :numbered:
 :experimental:
 
 include::attributes/attributes.adoc[]
 
-You can sync {HubName} to use the Red Hat Certified Collections available through your {PlatformNameShort} subscription or community collections available through {Galaxy}.
+Included with a subscription to {PlatformName}, {CertifiedColl} feature {CertifiedName} and {Valid}. 
+Using {HubNameMain}, your organization can access and curate a unique set of collections from all forms of Ansible content. 
```

Comment:
I created a google doc to see the whole doc better. If anything, you could look at the intro here:
https://docs.google.com/document/d/10sSlHpsk6Vj-IS3GIouUYrfDAnZWdhtY/edit#

Suggestion for first paragraph from google doc: 

{CertifiedColl} is included in your subscription to {PlatformName}. {CertifiedColl} includes two types of content: {CertifiedName} and {Valid}.

Not sure if "content" is the right word. Is it collections?